homeassistant/components/shelly/sensor.py

```diff
@@ -724,6 +760,11 @@ def native_value(self) -> StateType:
         """Return value of sensor."""
         return self.attribute_value
 
+    @property
+    def suggested_display_precision(self) -> int | None:
+        """Return the precision of the sensor."""
+        return self.entity_description.suggested_display_precision
+
```

Comment:
```suggestion
```


I don't think this is needed if its set in the description

https://github.com/home-assistant/core/blob/1759f58fc110b6c3ad81d859d6e5716182ad29d2/homeassistant/components/sensor/__init__.py#L405